### ISSUE TITLE
Not compatible with latest Flutter SDK

### ISSUE THREAD

**Issue by sbatezat** [Countly/countly-sdk-flutter-bridge#280]
Since I've updated my flutter version and the generated android project, Countly plugin doesn't works anymore

~I think it's related to     id "com.android.application" version "8.5.2" apply false~
It's related to Gradle 8, the default Gradle version on new Flutter projects : https://stackoverflow.com/a/77716017/863581

Here is the error message I've got :

> 
> * What went wrong:
> A problem occurred configuring project ':countly_flutter'.
> > Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
>    > Namespace not specified. Specify a namespace in the module's build file: /Users/me/pub-cache/git/countly-sdk-flutter-bridge-a70b062f35c1d88f99555b26e21fa45e556f4659/android/build.gradle. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.
> 
> If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

**Comment by turtledreams** [comment]
Hi @sbatezat. looks like it is asking for namespace. This was a fresh project or you updated your legacy project? Any AGP update happened?
Currently can you try the fix below and let us know if it works? You can add it directly to the local cache of the SDK and try to run the app again.
https://github.com/Countly/countly-sdk-flutter-bridge/pull/277/files

**Comment by sbatezat** [comment]
Hi @turtledreams , I can confirm it's solving the issue with your plugin.
For information, I've got the issue with other plugins, and we can fix it by adding that on our buld gradle, in the "allProjects" section, bellow repository declaration :


 ```
 subprojects {
        afterEvaluate { project ->
            if (project.hasProperty('android')) {
                project.android {
                    if (namespace == null) {
                        namespace project.group
                    }
                }
            }
        }
    }
```

**Comment by turtledreams** [comment]
Thank you for letting us know @sbatezat.  We will add the namespace in our next release to prevent this from happening. Will let you know after the release. Cheers.

**Comment by sbatezat** [comment]
Hi @turtledreams 
Is there something wrong with https://github.com/Countly/countly-sdk-flutter-bridge/pull/277 which is pending reviewer for two months?

**Comment by turtledreams** [comment]
Hi @sbatezat, no there is no problem, we are just waiting for a couple of updates to the underlying SDKs before we release a new version. It should happen soon (within a couple of weeks). Will keep you updated.

**Comment by sbatezat** [comment]
fixed in 24.11.0
